designs/waralee/src/waistband.mjs

```diff
@@ -48,13 +48,13 @@ function waraleeWaistband(
     case 'strapFront':
       partNr = 7
       waistBandLength -= waistBandLengthFront * 2
-      waistBandLength += options.knotInFront ? measurements.waist / 2 : 0
+      waistBandLength += options.knotPlacement == 'front' ? measurements.waist / 2 : 0
```

Comment:
`===` instead of `==` please 😊

It's a good habit to use strict comparison to avoid subtle type bugs, even if in this case there's no real risk.